rust/datafusion/src/physical_plan/mod.rs

```diff
@@ -62,7 +62,7 @@ pub trait ExecutionPlan: Debug + Send + Sync {
     /// downcast to a specific implementation.
     fn as_any(&self) -> &dyn Any;
     /// Get the schema for this execution plan
-    fn schema(&self) -> SchemaRef;
+    fn schema(&self) -> DFSchemaRef;
```

Comment:
When I was saying "physical plan doesn't use DFSchema` I guess I was imagining that `ExecutionPlan::schema()` continued to return `SchemaRef` -- there may be some reason that `ExecutionPlan` needs to return a DFSchema, but I think the design would be cleaner if we avoided this